api/control.proto

```diff
@@ -88,6 +88,7 @@ message ListNodesRequest {
 		map<string, string> labels = 3;
 		repeated NodeSpec.Membership memberships = 4;
 		repeated NodeRole roles = 5;
+		repeated string name_prefixes = 6 [(gogoproto.customname) = "NamePrefixes"];
```

Comment:
I know none of the fields around this are documented, but let's document this field.

```
// NamePrefixes will match all objects that match the given prefixes.
```